src/node.cc

```diff
@@ -2471,7 +2471,8 @@ void DLOpen(const FunctionCallbackInfo<Value>& args) {
                sizeof(errmsg),
                "The module '%s'"
                "\nwas compiled against the Node.js API. This feature is "
```

Comment:
Perhaps we should be even more verbose and say "... against the Node.js ABI-stable API." If you google "Node.js ABI-stable API", the first link is to nodejs/abi-stable-node, so this might be a good copy/paster for people looking up error messages.